blockonomics-woocommerce.php

```diff
@@ -627,6 +618,11 @@ function blockonomics_activation_hook() {
 // Page creation function  for the Blockonomics payement following woo-commerce page creation shortcode logic 
 function blockonomics_create_payment_page()
 {
+    global $wp_rewrite;
+
+    if ( null === $wp_rewrite ) {
+        $wp_rewrite = new \WP_Rewrite;
```

Comment:
Fix looks good, and works if [plugins_loaded](https://github.com/WordPress/WordPress/blob/master/wp-settings.php#L506) fires before the global variable [$wp_rewrite](https://github.com/WordPress/WordPress/blob/master/wp-settings.php#L544) exists. If the issue persists we can also consider hooking [wp_loaded](https://github.com/WordPress/WordPress/blob/master/wp-settings.php#L665), as this hook fires after wp_rewrite is set.